src/pint/toa.py

```diff
@@ -305,6 +308,9 @@ def get_TOAs(
     if usepickle and updatepickle:
         log.info("Pickling TOAs.")
         save_pickle(t, picklefilename=picklefilename)
+    if "pulse_number" in t.table.colnames and not include_pn:
+        log.warning(f"'pulse_number' column exists but not being read in")
```

Comment:
If the user has explicitly asked to ignore pulse numbers I don't think they should get a warning if pulse numbers are encountered. Nothing at all, or at most an info/debug-level note.

Reply:
ok, changed to debug